docs/tutorial/index.rst

```diff
@@ -200,11 +200,24 @@ If you now click on "View docs", you will see your documentation live!
 
    HTML documentation live on Read the Docs
 
+.. note::
+
+   If you don't see the ad, you might be using an ad blocker.
+   Our Ethical Ads network respects your privacy, doesn't target you,
+   and tries to be as unobstrusive as possible,
+   so we would like to kindly ask you to :doc:`not block us </advertising/ad-blocking>` |:heart:|
+
+   Advertisement is one of our main sources of revenue.
```

Comment:
I'd swap these two paragraphs. We don't want to lead w/ the ad blocking, people will stop reading if it doesn't apply to them.

Reply:
Good point, swapping 👍🏽